packages/office-addin-manifest/src/validate.ts

```diff
@@ -65,13 +66,24 @@ export async function validateManifest(
   manifestPath: string,
   verifyProduction: boolean = false
 ): Promise<ManifestValidation> {
-  try {
```

Comment:
Is there a reason we are removing the try/catch?

Reply:
We already have the try/catch on the `commands.ts` (the cli) for telemetry. So I thought this telemetry just had a lot of false positives because it keeps getting logged in our unit tests.
So now we just have the telemetry on the CLI part of the command

Reply:
This different is that the method can be called outside of the cli (like the test cases themselves do) and since the method is exported outside of the package then anyone can call this method as an API instead of via the cli.  We'd be losing all that telemetry as well.

Reply:
Yeah, we can keep this telemetry